Makefile

```diff
@@ -273,31 +276,31 @@ kind-down: $(KIND)
 # workaround for https://github.com/GoogleContainerTools/skaffold/issues/6416
 export SKAFFOLD_LABEL := skaffold.dev/run-id=gardener-local
```

Comment:
Shall we apply the same logic for the `SKAFFOLD_LABEL` env too? It looks like it is required for local provider context only too.